crates/bevy_time/src/common_conditions.rs

```diff
@@ -203,6 +203,31 @@ pub fn repeating_after_real_delay(
     }
 }
 
+/// Run condition that is active when the [`Time<Virtual>`] clock is paused.
+/// Use [`bevy_ecs::schedule::common_conditions::not`] to make it active when
```

Comment:
We should demonstrate how to use this too in the example :)

Reply:
Done.

I considered this before committing, but I thought it might be a bit much to show users how to use a different feature after already pointing it out in the description. But maybe more docs are better :)